python/cuml/test/test_svm.py

```diff
@@ -166,6 +166,14 @@ def compare_svm(svm1, svm2, X, y, b_tol=None, coef_tol=None,
             print("Skipping decision function test due to low  accuracy",
                   accuracy2)
 
+    # Compare support_ (dataset indicies of points that form the support vectors)
+    # and ensure that some overlap (~1/8) between two exists
```

Comment:
Is this 1/8 arbitrary or do we have some specific reason to think that's a safe value to test?

Reply:
1/8 is entirely arbitrary - I would have preferred to ensure a larger portion of the indices overlap, but one of the tests kept failing at 1/2 and 1/4!

I am not sure if this should be of concern, as I feel the differences in the indices might just be indicative of the difference between cuML's algorithm and that of sklearn.